server/src/concepts/authorization/controllers/auth.js

```diff
@@ -0,0 +1,74 @@
+import "dotenv/config";
+import jwt from "jsonwebtoken";
+
+const checkToken = (req, res) => {
+  let token = req.headers.token;
+  if (!token) {
+    token = req.body.token;
+  }
+  if (!token) return res.status(401).send("Access token is missing");
+
+  return checkToken;
```

Comment:
```suggestion
  return checkToken??;
```
what are you returning? function itself? This function should return a boolean:
- if token is good returns true
- if its wrong returns false
then in method when you are invoking it:

`if (!checkToken(req, res)) {
   return res.status(401)....
}`

Beside that I think it would be better to send a token to this function instead of req, res objects. This token you can define in each function when you would need that.

So function would look like this:
`checkToken(token)`